src/components/Shared/FlatCard.js

```diff
@@ -36,12 +37,16 @@ const FlatCard = props => {
         </CardContent>
       </CardActionArea>
       <CardActions>
-        <Button size="small" color="primary" component={Link} to={props.to}>
-          {t("buttons.edit")}
-        </Button>
-        <Button size="small" color="primary" onClick={props.onDelete}>
-          {t("buttons.delete")}
-        </Button>
+        {false && (
```

Comment:
How can this work - ie how is the second part ever evaluated?

Reply:
I disabled the action since we only have a "readonly" version.